back/app/services/xlsx_export/attendees_generator.rb

```diff
@@ -0,0 +1,16 @@
+module XlsxExport
+  class AttendeesGenerator < Generator
+    def generate_attendees_xlsx(users, view_private_attributes: false)
+      columns = [
```

Comment:
Do you think it could make sense to include an attended_at column?

Reply:
Not sure I understand. How would we know if someone attended an event?

Reply:
Does this export all attendees of an event? If so, I think the `Attendance` model could be used, although there doesn't seem to be an explicit attended_at timestamp. But probably this is really not that useful, it was just a small question.